DOCS-10012

domalessi opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
https://datadoghq.atlassian.net/browse/DOCS-10012

Changes "Service Catalog" to "Software Catalog" in text and images/vids.

### Merge instructions

<!-- 
If you're waiting for a release or there are other considerations that you want us to be aware of, list them here. 
If the PR is ready to be merged once it receives the required reviews, check the box below after you've created the PR.
-->

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

janine-c commented:
Hey Dom! I'm still looking through your content changes, but I wanted to flag that I investigated the link check failure, and it looks like a huge chunk of errors, if not all of them, come down to this folder needing to be renamed: https://github.com/DataDog/documentation/tree/domalessi/docs-10012/content/en/getting_started/service_catalog

It's a big PR so I wanted to give that feedback first! Hopefully this will help with that check so it doesn't block deployment. Will keep looking into the rest of the changes 🙂

janine-c reviewed (commented):
What a great job on a beast of a project!! I left some notes on anything I wasn't sure of, or some images that still had "Service Catalog" in them. Let me know if you want to chat about anything or if I can take another look or do anything else to help you get this over the finish line 😀

domalessi commented on content/en/software_catalog/use_cases/validating_service_definition.md:
```diff
@@ -15,12 +18,12 @@ further_reading:
   - link: "https://www.datadoghq.com/blog/manage-service-catalog-categories-with-service-definition-json-schema/"
     tag: "Blog"
-    text: "Manage Service Catalog entries with the Service Definition JSON Schema"
+    text: "Manage Software Catalog entries with the Service Definition JSON Schema"
```
```suggestion
    text: "Manage Service Catalog entries with the Service Definition JSON Schema"
```

domalessi replied:
```suggestion
    text: "Manage Service Catalog entries with the Service Definition JSON Schema"
```

domalessi commented on content/en/software_catalog/navigating.md:
```diff
@@ -1,35 +1,38 @@
 - link: "https://www.datadoghq.com/blog/manage-service-catalog-categories-with-service-definition-json-schema/"
   tag: "Blog"
-  text: "Manage Service Catalog entries with the Service Definition JSON Schema"
+  text: "Manage Software Catalog entries with the Service Definition JSON Schema"
```
```suggestion
  text: "Manage Service Catalog entries with the Service Definition JSON Schema"
```

domalessi commented on content/en/glossary/terms/service_map.md:
```diff
@@ -2,6 +2,6 @@
 core_product:
   - apm
-  - service catalog
+  - Software Catalog
```
```suggestion
  - software catalog
```

domalessi commented on content/en/software_catalog/use_cases/appsec_management.md:
```diff
@@ -0,0 +1,53 @@
+    text: "Datadog Application Security Management"
+---
+
+The Software Catalog enables organizations to seamlessly incorporate security into every development stage, ensuring a strong security posture across teams, applications, and systems.
+
+Software Catalog surfaces and centralizes security signals, enabling developers to prioritize actions and address vulnerabilities promptly. Meanwhile, managers can oversee risks, drive improvements, and ensure organizational compliance.
```
Great catch and yeah I noticed this too as I was going through! I think I'll note this to circle back to later but won't consider it a blocker for this particular PR.

domalessi commented on content/en/software_catalog/use_cases/_index.md:
```diff
@@ -0,0 +1,30 @@
+  - /service_catalog/use_cases
+---
+
+## Best practices guides
```
Yeah I've chatted with the PMs and the goal is actually to get rid of this page and fold the content into existing documentations -- I'll do that in a separate PR!

domalessi commented on content/en/software_catalog/service_definitions/v3-0.md:
```diff
@@ -15,7 +17,7 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/service-catalog-schema-v3/"
   tag: "Blog"
-  text: "Improve developer experience and collaboration with Service Catalog schema v3.0"
+  text: "Improve developer experience and collaboration with Software Catalog schema v3.0"
```
```suggestion
  text: "Improve developer experience and collaboration with Service Catalog schema v3.0"
```

domalessi commented on content/en/software_catalog/navigating.md:
```diff
@@ -21,15 +24,15 @@ further_reading:
+  text: "I use GitHub Ac­tions for Data­dog's Software Catalog, and you should, too"
 algolia:
-  tags: ['service catalog']
+  tags: ['Software Catalog']
```
```suggestion
  tags: ['software catalog']
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -1,53 +1,58 @@
   text: "Investigate memory leaks and OOMs with Datadog's guided workflow"
 algolia:
-  tags: ['service catalog']
+  tags: ['Software Catalog']
```
```suggestion
  tags: ['software catalog']
```

domalessi commented on content/en/software_catalog/manage.md:
```diff
@@ -1,74 +1,79 @@
+  text: "Datadog Software Catalog"
 algolia:
-  tags: ['service catalog']
+  tags: ['Software Catalog']
```
```suggestion
  tags: ['software catalog']
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -1,53 +1,58 @@
-Datadog [Service Catalog][1] provides a consolidated view of your services, combining ownership metadata, performance insights, security analysis, cost allocation, and much more. It makes it easy for organizations to achieve end-to-end service ownership at scale, get real-time performance insights, detect and address reliability and security risks, and manage application dependencies all in one p ...
+The Datadog [Software Catalog][1] provides a centralized, dynamic view of your software ecosystem, enabling teams to track ownership, monitor performance, manage dependencies, and enforce security and compliance standards—all in one place.
+
+Built on real-time telemetry and automated metadata collection, Datadog Software Catalog extends beyond traditional service catalogs by offering integrations with observability, security, and cost management tools. It empowers engineering, SRE, security, and platform teams to maintain visibility, streamline operations, and ensure service reliability at scale.
```
```suggestion
Built on real-time telemetry and automated metadata collection, Datadog Software Catalog extends beyond traditional software catalogs by offering integrations with observability, security, and cost management tools. It empowers engineering, SRE, security, and platform teams to maintain visibility, streamline operations, and ensure service reliability at scale.
```

domalessi commented on content/en/software_catalog/manage.md:
```diff
@@ -1,74 +1,79 @@
 - link: "https://www.datadoghq.com/blog/manage-service-catalog-categories-with-service-definition-json-schema/"
   tag: "Blog"
-  text: "Manage Service Catalog entries with the Service Definition JSON Schema"
+  text: "Manage Software Catalog entries with the Service Definition JSON Schema"
```
```suggestion
  text: "Manage Service Catalog entries with the Service Definition JSON Schema"
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -1,53 +1,58 @@
 - link: "https://www.datadoghq.com/blog/manage-service-catalog-categories-with-service-definition-json-schema/"
   tag: "Blog"
-  text: "Manage Service Catalog entries with the Service Definition JSON Schema"
+  text: "Manage Software Catalog entries with the Service Definition JSON Schema"
```
```suggestion
  text: "Manage Service Catalog entries with the Service Definition JSON Schema"
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -23,31 +26,33 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/service-catalog-schema-v3/"
   tag: "Blog"
-  text: "Improve developer experience and collaboration with Service Catalog schema version 3.0"
+  text: "Improve developer experience and collaboration with Software Catalog schema version 3.0"
```
```suggestion
  text: "Improve developer experience and collaboration with Service Catalog schema version 3.0"
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -23,31 +26,33 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/service-ownership-best-practices-datadog/"
   tag: "Blog"
-  text: "Best practices for end-to-end service ownership with Datadog Service Catalog"
+  text: "Best practices for end-to-end service ownership with Datadog Software Catalog"
```
```suggestion
  text: "Best practices for end-to-end service ownership with Datadog Service Catalog"
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -23,31 +26,33 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/shift-left-datadog-service-catalog/"
   tag: "Blog"
-  text: "Improve your shift-left observability with the Datadog Service Catalog"
+  text: "Improve your shift-left observability with the Datadog Software Catalog"
```
```suggestion
  text: "Improve your shift-left observability with the Datadog Service Catalog"
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -23,31 +26,33 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/github-actions-service-catalog/"
   tag: "Blog"
-  text: "I use GitHub Ac­tions for Data­dog's Service Catalog, and you should, too"
+  text: "I use GitHub Ac­tions for Data­dog's Software Catalog, and you should, too"
```
```suggestion
  text: "I use GitHub Ac­tions for Data­dog's Service Catalog, and you should, too"
```

domalessi commented on content/en/software_catalog/_index.md:
```diff
@@ -23,31 +26,33 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/service-catalog-setup/"
   tag: "Blog"
-  text: "Easily add tags and metadata to your services using the simplified Service Catalog setup"
+  text: "Easily add tags and metadata to your services using the simplified Software Catalog setup"
```
```suggestion
  text: "Easily add tags and metadata to your services using the simplified Service Catalog setup"
```

domalessi commented on static/images/tracing/software_catalog/service_catalog_definition_yaml.png:
Some of the images I duplicated and renamed the dupe to be "software catalog" so we have both versions. Per guidance here: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1985314888/Screenshots+and+Videos#Update-Image-Repository

domalessi commented:
@janine-c You are seriously the most amazing human. Thank you SO MUCH. 

I've addressed all of your feedback:
- I tried to be diligent about making sure blog _links_ weren't changed, but I totally slipped on checking the blog names! Did another pass through to change "Software Catalog" to "Service Catalog" in all blog titles. Probably still somewhat confusing but that'll just have to be the state of the world until the Blog articles are updated. 
- I blurred out PII in screenshots and videos. Note that there are some duplicate and unused images/videos. I was wary about deleting things and often just duplicated and renamed the thing. I did some clean up but there may be a few stragglers. In any case, I confirmed that any _used_ images/vids are all looking good. (There is one exception where there's a small amount of text that reads "Service Catalog" but I'm not considering this a blocker and have noted it to circle back on later.)
- I've confirmed that the Glossary looks good locally -- it now shows "Software Catalog" on the Service and Service Map entries.
- I've updated casing where needed -- thank you so much for catching that omg. 
- I saw your note about the Best Practices page. I've chatted with my PMs about this already and will address in a separate PR!
- Also saw your note about inconsistencies around using "Software Catalog" vs "The Software Catalog". I won't block on that for this PR but will note it to come back to later!
- I've also updated the getting_started/service_catalog folder to getting_started/software_catalog and I've confirmed that the former redirects to the latter. 

I think that's everything! 

Thank you again for taking on this beast of a review and catching a lot of tiny details that I overlooked in my frenzy.

janine-c reviewed (approved):
You're an absolute HERO for getting this all done, Dom!! I checked the files you edited since my last review and they all look fantastic. I can see that there's one merge conflict waiting - hope that isn't too hard to resolve. So glad I could help you with this one, and hope the last few steps to get it out the door are nice and smooth!

timsara331 reviewed (approved):
LGTM

domalessi commented:
/merge
